Analytical_Engine/Query/Graph/Relation.cs

```diff
@@ -75,6 +93,12 @@ public static List<IRelation> Relation(this Graph graph, IBHoMObject source, IBH
         [Output("relations", "The collection of matching IRelation Guids.")]
         public static List<Guid> Relation(this Graph graph, Guid source, Guid target, RelationDirection relationDirection = RelationDirection.Forwards)
         {
+            if (graph == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the relations of a null graph.");
+                return null;
```

Comment:
Why not return an empty collection to follow the pattern?
```suggestion
                return new List<Guid>();
```